service/tasks/driver.py

```diff
@@ -847,11 +847,10 @@ def _deploy_ready_failed_email_test(
 
 @task(name="deploy_ready_test",
       default_retry_delay=64,
-      # 16 second hard-set time limit. (NOTE:TOO LONG? -SG)
       soft_time_limit=120,
-      max_retries=300  # Attempt up to two hours
+      max_retries=115  # Attempt up to two hours
```

Comment:
Could you comment on these time values? Confused about the range of possibilities this enables, like whats the slowest each could take * 115

Reply:
Is the comment wrong?

Reply:
Looks like the comment used to be wrong when `max_retries` was 300. It's now correct. :)